hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3,18 +3,1146 @@
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_spi_device_eeprom
+      desc: '''Verify the SPI device in EEPROM mode.
+
+            SW puts the SPI device in EEPROM mode and allows a firmware image (a.k.a. a known
+            payload) to be downloaded into the in-built EEPROM. SW verifies the integrity of the
+            payload upon reception by reading the EEPROM. Details TBD since the feature is NA yet.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // SPI_HOST (pre-verified IP) integration tests:
+    {
+      name: chip_spi_host_tx_rx
+      desc: '''Verify the transmission of data on the chip's SPI host port.
+
+             Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_spi_pass_through
+      desc: '''Verifies the pass through mode from an end-to-end perspective.
+
+            SW configures the SPI device and host in pass through mode. The testbench sends a random
+            payload over the SPI device interface and verifies its integrity on the SPI host
+            interface.  Details TBD.
+      '''
+      milestone: V2
+      tests: []
+    }
+
+    // I2C (pre-verified IP) integration tests:
+    {
+      name: chip_i2c_host_tx_rx
+      desc: '''Verify the transmission of data over the chip's I2C host interface.
+
+            The SW test writes a known data over the chip's I2C host interface, which is verified by
+            the testbench (which acts as the I2C device). Likewise, SW test then reads and verifies
+            a known data. SW validates the reception of RX watermark, FMT overflow, RX overflow,
+            NAK, FMT watermark and trans complete interrupts (the SW test / testbench work together
+            to create those scenarios).
+            Verify all instances of I2C in the chip independently.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_i2c_device_tx_rx
+      desc: '''Verify the transmission of data over the chip's I2C device interface.
+
+            The testbench writes a known data over the chip's I2C device interface, which is
+            verified by the SW test for correctness. Testbench then reads and verifies
+            a known data. SW validates the reception of TBD interrupts (the SW test / testbench
+            create those scenarios).
+            Verify all instances of I2C in the chip independently.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // USB (pre-verified IP) integration tests:
+    {
+      name: chip_usb_fs_tx_rx
+      desc: '''Verify the transmission of data over the USB at full speed.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_usb_vbus
+      desc: '''Verify that the USB device can detect the presence of VBUS from the USB host.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
```

Comment:
we may want to long term consider grouping this test with the chip wide low power test.